lib/friendly_shipping/services/ups_json/parse_rates_response.rb

```diff
@@ -32,84 +32,71 @@ def build_rates(rates_result, shipment)
               total_cost = ParseMoneyHash.call(rated_shipment['TotalCharges'], 'TotalCharges')
               insurance_price = ParseMoneyHash.call(rated_shipment['ServiceOptionsCharges'], 'ServiceOptionsCharges')
               negotiated_rate = ParseMoneyHash.call(rated_shipment.dig('NegotiatedRates', 'NetSummaryCharges', 'GrandTotal'), 'GrandTotal')
+              negotiated_charges = ParseMoneyHash.call(rated_shipment.dig('NegotiatedRates', 'ItemizedCharges'), 'ItemizedCharges')
+
+              itemized_charges = rated_shipment['ItemizedCharges'].is_a?(Hash) ? [rated_shipment['ItemizedCharges']] : rated_shipment['ItemizedCharges']
```

Comment:
You can use `Array()` here.

Reply:
It is actually Array.wrap() that is my friend here. I was a little worried about pulling in ActiveSupport just for this but it was already in.

Reply:
Sure, that works. `Array()` is provided by Ruby's `Kernel` class and does the same thing with a few [minor differences](https://apidock.com/rails/v5.2.3/Array/wrap/class).

Reply:
The Kernel method does an undesirable to_a on the passed in hash which the ActiveSupport method explicitly avoids.

Reply:
Oh, that makes sense then. 👍